aion-api/src/main/java/com/aion/dashboard/services/ThirdPartyService.java

```diff
@@ -33,7 +33,7 @@ public String getCirculatingSupply() throws IOException {
             List<CirculatingSupply> circulatingSupplies = circulatingSupply.findAll();//this fails on test nets
             TreeMap<Instant, BigInteger> temp= new TreeMap<>(Instant::compareTo);
             for (CirculatingSupply supply : circulatingSupplies) {//read the table and store the contents in memory
-                temp.put(supply.getStartDate().toInstant(), supply.getSupply().toBigInteger());
+                temp.put(supply.getTimestamp().toInstant(), supply.getSupply().toBigInteger());
```

Comment:
Not sure I fully follow the intention behind this endpoint, when does this cache get invalidated? (It seems like circulatingSupplyMap will get instantiated on the first call, then never be null again correct?)

Reply:
I removed this class, with the new implementation it got deprecated